1001/cpp17.cpp

```diff
@@ -0,0 +1,27 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+/**
+ * Returns  the number of problems stored in each computer.
+ * @param n denotes the total number of problems.
+ */
+vector<int> problemsStored(int n){
+    // Implement this method
+
+}
+
+/**
+ * Takes care of the problem input and output.
+ */
+int main(){
+    int T;
+    scanf("%d", &T);
+    while(T--){
```

Comment:
use a for loop, with `caseno`.

Also, the c++ styling standard wise, use space after while, if, for, etc.